src/constants/assets.ts

```diff
@@ -1,2 +1,42 @@
+import { ICP_CANISTER_ID, WICP_CANISTER_ID, XTC_CANISTER_ID } from './canister';
+
 export const E8S_PER_ICP = 100000000;
 export const CYCLES_PER_TC = 1000000000000;
+export const USD_PER_TC = 1.42656;
+
+export const TOKEN_IMAGES = {
+  XTC: 'xtc',
+  ICP: 'dfinity',
+  WICP: 'wicp',
+};
+
+// Is this const OK? We should use keyring.getTokenInfo() instead.
```

Comment:
we should honestly. But this could serve as a default backup to prevent anything from breaking